src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -383,6 +384,12 @@ record = _serverStore.Cluster.ReadDatabase(context, Name, out index);
                 }, DatabaseShutdown);
                 _serverStore.LicenseManager.LicenseChanged += LoadTimeSeriesPolicyRunnerConfigurations;
                 IoChanges.OnIoChange += CheckWriteRateAndNotifyIfNecessary;
+
+                using (DocumentsStorage.ContextPool.AllocateOperationContext(out DocumentsOperationContext ctx))
```

Comment:
let's move this before `_clusterTransactionsTask`

Reply:
Done